pkgs/servers/monitoring/uptime-kuma/default.nix

```diff
@@ -0,0 +1,55 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, fetchurl, substituteAll, iputils }:
+let
+  deps = import ./composition.nix { inherit pkgs; };
+
+  dist = stdenv.mkDerivation rec {
+    pname = "uptime-kuma-dist";
+    version = "1.17.1";
```

Comment:
Please use the same version across the two packages.

Reply:
Done !